pom.xml

```diff
@@ -106,7 +112,8 @@
       <version>8.2.0</version>
       <scope>test</scope>
     </dependency>
-  </dependencies>
+   </dependencies>
+  </dependencyManagement>
 
   <build>
     <plugins>
```

Comment:
I may add a Maven profile to not execute the `maven-gpg-plugin` and `nexus-staging-maven-plugin  plugins outside of a release process (as done [here](https://github.com/quick-perf/quickperf/blob/c25891ffd45977d5e681702972b557e6ad4de170/pom.xml#L198))